containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
I tried this with the override file and also got the “Error establishing a database connection” message. After a bit of config wrangling, I got things working by falling back to my trusty `env` file in the override file:

```yaml
mysql:
  environment:
    - MYSQL_ROOT_PASSWORD=root
    - MYSQL_DATABASE=${MYSQL_DATABASE}
    - MYSQL_USER=${MYSQL_USER}
    - MYSQL_PASSWORD=${MYSQL_PASSWORD}
  ports:
    - 3306:3306
wordpress:
  build: ./containers/${MYSQL_DATABASE}
  environment:
    - WORDPRESS_DB_NAME=${MYSQL_DATABASE}
    - WORDPRESS_DB_USER=${MYSQL_USER}
    - WORDPRESS_DB_PASS=${MYSQL_PASSWORD}
  ports:
    - 80:80
```